requirements.txt

```diff
@@ -0,0 +1,29 @@
+#
+# This file is autogenerated by pip-compile with python 3.9
+# To update, run:
+#
+#    pip-compile
+#
+cerberus==1.3.4
+    # via -r requirements.in
+certifi==2021.5.30
+    # via elasticsearch
+elasticsearch==7.13.3
+    # via -r requirements.in
+nmslib==2.1.1
+    # via -r requirements.in
+numpy==1.21.0
+    # via
+    #   -r requirements.in
```

Comment:
I believe you have to checkin requirements.in

Reply:
Done